dist/templates/ovn-ipsec.yaml.j2

```diff
@@ -230,6 +230,12 @@ spec:
           # Workaround for https://github.com/libreswan/libreswan/issues/373
           ulimit -n 1024
 
+          # Remove stale pluto.pid if no pluto process is running
+          if [ -f /var/run/pluto/pluto.pid ] && ! pgrep -F /var/run/pluto/pluto.pid > /dev/null 2>&1; then
+            echo "Removing stale pluto.pid"
+            rm -f /var/run/pluto/pluto.pid
```

Comment:
sorry not familiar with this code but i see youre removing the pid file in two places - here and the daemonset - can you explain more why you did that?

Reply:
when `ovn-ipsec` pod is restarted, then pluto is killed, but its pid file on the host is not deleted timely before new ovn-ipsec container comes up, so it's failing to spin up new pluto process because old pid file is still present. so this code deletes stale pid file if present.

The daemonset.yaml is a template file for helm based deployment, so we needed this change at two places.

Reply:
I see - thanks. Are you only seeing the stale pid if the node is force rebooted / power off scenarios or?